src/test/java/teammates/test/cases/util/FieldValidatorTest.java

```diff
@@ -367,9 +368,15 @@ public void invalidityInfoFor_validGender_returnEmptyString() {
     public void invalidityInfoFor_invalidGender_returnErrorString() {
```

Comment:
Edit `nationality`? The test case added in FieldValidator was to test that the error message is properly sanitized.